fathom/src/surface/elaboration.rs

```diff
@@ -118,27 +118,27 @@ impl<'arena> RigidEnv<'arena> {
-        const VAR2: core::Term<'_> = core::Term::RigidVar(env::LocalVar::last().prev().prev());
-        const VAR3: core::Term<'_> =
-            core::Term::RigidVar(env::LocalVar::last().prev().prev().prev());
-        const UNIVERSE: core::Term<'_> = core::Term::Universe;
-        const FORMAT_TYPE: core::Term<'_> = core::Term::Prim(Prim::FormatType);
-        const BOOL_TYPE: core::Term<'_> = core::Term::Prim(Prim::BoolType);
-        const U8_TYPE: core::Term<'_> = core::Term::Prim(Prim::U8Type);
-        const U16_TYPE: core::Term<'_> = core::Term::Prim(Prim::U16Type);
-        const U32_TYPE: core::Term<'_> = core::Term::Prim(Prim::U32Type);
-        const U64_TYPE: core::Term<'_> = core::Term::Prim(Prim::U64Type);
-        const S8_TYPE: core::Term<'_> = core::Term::Prim(Prim::S8Type);
-        const S16_TYPE: core::Term<'_> = core::Term::Prim(Prim::S16Type);
-        const S32_TYPE: core::Term<'_> = core::Term::Prim(Prim::S32Type);
-        const S64_TYPE: core::Term<'_> = core::Term::Prim(Prim::S64Type);
-        const ARRAY8_TYPE: core::Term<'_> = core::Term::Prim(Array8Type);
-        const ARRAY16_TYPE: core::Term<'_> = core::Term::Prim(Array16Type);
-        const ARRAY32_TYPE: core::Term<'_> = core::Term::Prim(Array32Type);
-        const ARRAY64_TYPE: core::Term<'_> = core::Term::Prim(Array64Type);
-        const POS_TYPE: core::Term<'_> = core::Term::Prim(Prim::PosType);
+        const VAR0: Term<'_> = Term::RigidVar(Span::Empty, env::LocalVar::last());
+        const VAR1: Term<'_> = Term::RigidVar(Span::Empty, env::LocalVar::last().prev());
+        const VAR2: Term<'_> = Term::RigidVar(Span::Empty, env::LocalVar::last().prev().prev());
+        const VAR3: Term<'_> =
+            Term::RigidVar(Span::Empty, env::LocalVar::last().prev().prev().prev());
+        const UNIVERSE: Term<'_> = Term::Universe(Span::Empty);
+        const FORMAT_TYPE: Term<'_> = Term::Prim(Span::Empty, FormatType);
+        const BOOL_TYPE: Term<'_> = Term::Prim(Span::Empty, BoolType);
+        const U8_TYPE: Term<'_> = Term::Prim(Span::Empty, U8Type);
+        const U16_TYPE: Term<'_> = Term::Prim(Span::Empty, U16Type);
+        const U32_TYPE: Term<'_> = Term::Prim(Span::Empty, U32Type);
+        const U64_TYPE: Term<'_> = Term::Prim(Span::Empty, U64Type);
+        const S8_TYPE: Term<'_> = Term::Prim(Span::Empty, S8Type);
+        const S16_TYPE: Term<'_> = Term::Prim(Span::Empty, S16Type);
+        const S32_TYPE: Term<'_> = Term::Prim(Span::Empty, S32Type);
+        const S64_TYPE: Term<'_> = Term::Prim(Span::Empty, S64Type);
+        const ARRAY8_TYPE: Term<'_> = Term::Prim(Span::Empty, Array8Type);
+        const ARRAY16_TYPE: Term<'_> = Term::Prim(Span::Empty, Array16Type);
+        const ARRAY32_TYPE: Term<'_> = Term::Prim(Span::Empty, Array32Type);
+        const ARRAY64_TYPE: Term<'_> = Term::Prim(Span::Empty, Array64Type);
+        const POS_TYPE: Term<'_> = Term::Prim(Span::Empty, PosType);
```

Comment:
Eventually would be cool if these could point to some place in a `prims` module.